temoa-uiuc/Snakefile

```diff
@@ -14,10 +14,6 @@ DBS = ["UIUC_2050_cost"]
 SCENARIOS = ["costScenario"]
 FOLDER = ["UIUC_2050_cost_costScenario_model"]
 
```

Comment:
```suggestion
# uncomment these lines to run the land minimization scenario.

# DBS = ["UIUC_2050_land"]	
# SCENARIOS = ["landScenario"]	
# FOLDER = ["UIUC_2050_land_landScenario_model"]
```

Reply:
I added a comment explaining the purpose of those lines.

At the moment, we are making changes to one scenario at a time. Not a good idea to delete them right now. 

Instead, we should change the ``# All Scenarios`` block, because none of those will be a part of the final analysis.
If we *must* delete comments, then remove the ``All Scenarios`` block.

Reply:
Surely snakemake has a more graceful way to do this?

Reply:
I think that is the way to do it, based on the way I've written the rules. 
You can limit the rules that are executed with the ``--until`` flag, but the current rules will run every scenario listed in the DB, SCENARIO, and FOLDER variables.

The only alternative I can think of is to write one rule for each scenario, but then the snakefile is repeating code. This was the approach I took in the snakefile for ``cairo``, but each scenario actually has a different command line argument and snakemake doesn't like it when the wildcards aren't the same.